ci-operator/step-registry/openshift/e2e/openstack/kuryr-migration/openshift-e2e-openstack-kuryr-migration-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
+  as: openshift-e2e-openstack-kuryr-migration
+  steps:
+    pre:
+      - chain: ipi-openstack-pre-noramfs
+    test:
+      - ref: openstack-kuryr-migration
+      - chain: openstack-test-e2e
+    post:
+      - chain: ipi-openstack-post
+    env:
+      CONFIG_TYPE: "minimal"
```

Comment:
```suggestion
      CONFIG_TYPE: "proxy"
```

Reply:
@mandre why do you want it to be "proxy"? I think "minimal" is ok.

Reply:
I don't think we need the job to be proxy, unless the environment requires it (the above comment is quite old, and the patch was quite different at the time).